content/en/os/1.15.x/api/settings/aws/_index.markdown

```diff
@@ -0,0 +1,11 @@
++++
+title="aws"
+type="docs"
+toc_hide=true
+description="Settings specific to the AWS platform (`settings.aws.*`)"
++++
+
+Only pertinent on `aws-*` variants, these settings are automatically gathered using instance metadata.
```

Comment:
This is not actually accurate anymore. With [support added for IAM Roles Anywhere](https://github.com/bottlerocket-os/bottlerocket/pull/2553/files), even the VMware and Metal variants can use this in combination with the k8s ECR credential provider.

Reply:
TIL. I'll revise!

Reply:
Fixed and adding cross-linking with with the setting in in `settings.kubernetes.credential-providers` (and fixed some formatting along the way).